docs/js-core-1/week-2/lesson.md

```diff
@@ -429,6 +434,59 @@ So to sum up, `for of` loops are pretty similar to `for` loops, but:
 
 Next week, we will think more about the most common iterable object in JavaScript: [arrays](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array).
 
+## PD (Dealing with conflict in teams)
+
+Adapted from [Tectonica Interpersonal Conflict Resolution](https://github.com/Techtonica/curriculum/blob/main/career/conflict-resolution.md).
+
+**Session objective:** This session explores what conflict is and the common ways we address it; how can we add to our conflict resolution toolkits and become better at solving conflict effectively; what effective body language is; what effective choice of words is; and how to build trust among teams.
```

Comment:
This sentence, which is 47 words long, is written at a postgraduate level. A majority of our trainees are not yet reading English at this level.

Can you try again here? Use Hemingway to help you revise this. Break up your sentences. Use bullet points if it helps! https://hemingwayapp.com/

Reply:
I had a go.